src/components/core/GeoChart.tsx

```diff
@@ -0,0 +1,122 @@
+// (C) 2019 GoodData Corporation
+import * as React from "react";
+import { WrappedComponentProps } from "react-intl";
+import { Execution } from "@gooddata/typings";
+import * as cx from "classnames";
```

Comment:
it could be :)
```
import * as React from "react";
import * as cx from "classnames";
import { WrappedComponentProps } from "react-intl";

import { Execution } from "@gooddata/typings";
```